degruyter-acs-numbered-2.csl

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <info>
+    <title>De Gruyter ACS Numbered</title>
+    <title-short>DGACS</title-short>
+    <id>http://www.zotero.org/styles/degruyter-acs-numbered-2</id>
+    <link href="http://www.zotero.org/styles/degruyter-acs-numbered-2" rel="self"/>
+    <author>
```

Comment:
Please remove the authors other than the one making the current new style (and list them as 'author')